website/data/front/conformance.mdx

```diff
@@ -0,0 +1,15 @@
+import ConformanceShowcase from "../../src/components/ConformanceShowcase";
+
+# FDC3 Conformance
+
+FDC3 has a [conformance framework](https://github.com/finos/FDC3-Conformance-Framework) that works to ensure compatibility with the standard. 
+
+Check the [community page](/community) for details of who has passed the conformance tests.
+
+<img className="front-conformance-logo" src="/img/community/certified-1.2.png" />
```

Comment:
**a11y**

So, this image needs alt text. But also, it kind of serves as a heading. Maybe consider something like this:

```
<img className="front-conformance-logo" role="heading" aria-level="3" aria-label="FDC3 1.2 Certified Desktop Agents" />
```

It may also be worth just doing a separate PR to clean up some accessibility things, like how the home page uses H1's for every heading instead of only 1.